packages/design-system-docs/src/scripts/components/PageBlock.jsx

```diff
@@ -42,6 +43,24 @@ class PageBlock extends React.PureComponent {
     );
   }
 
+  statusBadge() {
+    if (this.props.status) {
+      const classes = classNames(
+        'ds-c-badge ds-u-margin-right--1 ds-u-text-transform--capitalize',
+        {
+          'ds-c-badge--success': this.props.status === 'Ready',
```

Comment:
I'm wondering if we should update these colors too. I don't think we've ever used ready, and we should add a color for Deprecated which is just the default grey right now. Maybe "Draft" can be `ds-c-badge--warn` and "Deprecated" can be `ds-c-badge--alert`.
This should be updated in `statusBadge()` line 12 in `PageHeader.jsx` as well. I would even add a comment to both instances of `statusBadge()` explaining how they are both copies of the same code so in the future we wont forget to change both.